docs/addon-kubeflow.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: docs
+title: "Kubeflow addon"
+permalink: /docs/addon-kubeflow
+---
+
+<span class="p-label--new">New in 1.16.2 </span>
+# Kubeflow add-on
+
+Working with machine learning pipelines can be a complex and challenging task.
+[Kubeflow][kubeflow] is an up-and-coming solution that aims to tackle that
+complexity and make it easy to deploy machine learning pipelines. You can try
+it out easily with Microk8s.
+
+Kubeflow is a collection of microservices that integrates many different machine
+learning technologies under a single, deployable umbrella. The name comes from
+combining Kubernetes and TensorFlow, and unsurprisingly, it includes a lot of
+TensorFlow functionality right out of the box, such as model training and
+serving. It comes with support for Jupyter Notebooks for interactively
+exploring your data and adds in support for pipelines, backed by
+Argo Workflows. There's many more services included; for an up-to-date list,
+check out <https://www.kubeflow.org/docs/components/>.
+
+## Using Kubeflow in Microk8s
+
+The Kubeflow add-on was released in the middle of the 1.16.x series of
+Kubernetes. To avoid potentially breaking earlier installs, only
+versions of MicroK8s installed using the latest stable release will include
+this add-on, until the release of 1.17. You will need to have installed
+MicroK8s like this:
+
+```bash
+sudo snap install microk8s --classic
+```
+
+Kubeflow is then enabled by running:
+
+```bash
+sudo microk8s.enable kubeflow
+```
+
+Note that this will also enable some additional add-ons required by Kubeflow.
+The process may take several minutes to fetch and install all the required
+components.
+
+Once the add-ons have been successfully installed, the Kubeflow dashboard URL
+will be output. You can open this on your browser, but you will need the
+autogenerated username and password. Obtain these by running:
+
+```bash
+microk8s.juju config ambassador-auth username
+microk8s.juju config ambassador-auth password
+```
+
+You should then see a page that looks like this:
+
+**IMAGE**
+
```

Comment:
It would be good to leave with a link for further help. I took a look at the kubeflow site, but I'm not sure if you have a suggestion

Reply:
We could recommend the Kubeflow docs, though there isn't a best link to put here. Maybe just https://www.kubeflow.org/docs/about/? Alternatively, we could recommend opening an issue in https://github.com/juju-solutions/bundle-kubeflow/